components/staticFeeds/about-frame.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
This has a different structure than the filtered-resources/about-frame.json, may need updating?

Reply:
Reverted the change to filtered-resources/about-frame.json so they are now identical